algorithmForNN/workWithPointCloud.py

```diff
@@ -0,0 +1,198 @@
+import open3d as o3d
+import numpy as np
+import os
+import math
+
+from workWithFile import write_zeros_in_file, write_ones_to_file_by_index_list
+
+def get_point_cloud_from_bin_file(path_to_bin_file : str) -> o3d.geometry.PointCloud:
+    '''
+    Function allows you to get a point cloud from a file with lidar data 
+    '''
+    point_cloud_np = np.fromfile(path_to_bin_file, dtype=np.float32).reshape(-1, 4)[:, :3]
+    point_cloud = o3d.geometry.PointCloud()
+    point_cloud.points = o3d.utility.Vector3dVector(point_cloud_np[:, :3])
+    return point_cloud
+
+def convert_point_cloud_to_numpy_array(point_cloud : o3d.geometry.PointCloud) -> np.ndarray:
+    '''
+    Function converts a point cloud to numpy array
+    '''
+    return np.asarray(point_cloud.points)
+
+def convert_numpy_to_point_cloud(numpy_array : np.ndarray) -> o3d.geometry.PointCloud:
```

Comment:
previous method has "array" word in the name, but this doesn't -- it is better to choose one notation